muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/communicator/CommunicatorController.java

```diff
@@ -688,5 +693,18 @@ private boolean isActiveUser(UserSchoolDataIdentifier userSchoolDataIdentifier)
     }
     return tagsStr;
   }
+  
+  // Vacation notifications for auto reply
 
+  public VacationNotifications createVacationNotification(UserEntity sender, UserEntity receiver, Date notificationDate) {
+    return vacationNotificationsDAO.create(sender, receiver, notificationDate);
+  }
+  
+  public VacationNotifications updateVacationNotification(VacationNotifications vacationNotification,Date notificationDate) {
```

Comment:
Basically nitpicking but the method could be updateVacationNotificationDate to better imply that the whole object is not getting updated; just its date field.